CMakeLists.txt

```diff
@@ -109,6 +109,7 @@ include_directories(${EIGEN_DIR})
 
 # See https://gitlab.com/libeigen/eigen/-/blob/master/COPYING.README
 add_definitions(-DEIGEN_MPL2_ONLY)
+add_definitions(-DEIGEN_DONT_PARALLELIZE)
```

Comment:
Can you please add this to the flags used by the R package as well?

https://github.com/microsoft/LightGBM/blob/44fe591a60d7427d64997c37b22768a92c97c47b/R-package/configure.ac#L38

https://github.com/microsoft/LightGBM/blob/44fe591a60d7427d64997c37b22768a92c97c47b/R-package/configure.win#L22

Reply:
For non-Windows, we have a comment-triggered CI job that will update `configure` based on changes to `configure.ac`. I'll trigger that job right now.

For `configure.win`, nothing needs to be regenerated. `configure.win` is executed directly, instead of being used as a template.

These things are documented at https://github.com/microsoft/LightGBM/tree/master/R-package#changing-the-cran-package but that README is fairly large so it's easy to miss.